src/models/common/training.py

```diff
@@ -0,0 +1,57 @@
+import tensorflow as tf
+import os
+import time
+
+class Trainer():
+    def __init__(self, dataset, hparams):
+        self.dataset = dataset
+        self.hparams = hparams
+        self.step = tf.Variable(0)
+
+    def init_checkpoint(self, ckpt):
+        self.ckpt = ckpt
+        self.manager = tf.train.CheckpointManager(self.ckpt,
+                                                  os.path.join(self.hparams['save_dir'], 'ckpts', self.hparams['name']),
+                                                  max_to_keep=3)
+
+        self.ckpt.restore(self.manager.latest_checkpoint)
+        if self.manager.latest_checkpoint:
```

Comment:
I think the meaning would be clearer with an `is not None` here (assuming that what it's doing).

Reply:
Not sure if it checks for `None`, this is from tensorflow documentation.
Also not related to pull